1.24/dev/development_gitpod.html

```diff
@@ -464,6 +464,7 @@ <h2>Gitpod<a class="headerlink" href="#gitpod" title="Permalink to this heading"
 environment as code and start instant and fresh development environments for
 each new task directly from your browser. This reduces the need to install local
 development environments and deal with incompatible dependencies.</p>
+ <p><span class="versionmodified deprecated">Deprecated since version 1.24: </span>The Gitpod instance is no longer maintained </p>
```

Comment:
```suggestion
 <p><span class="versionmodified deprecated">Deprecated since version 1.24: </span>The Gitpod instance is no longer maintained. </p>
```